.github/ISSUE_TEMPLATE/issue_template.md

```diff
@@ -0,0 +1,13 @@
+## Expected Behavior
+
+##Actual Behavior
+
+##Steps to Reproduce the Problem
+1.
+2.
+3.
+
+##Specifications
```

Comment:
In my opinion `Platform` and `Subsystem` are not helpful information. Instead that we can ask about version  of `python` / `docker`.

Reply:
added requested changes. Thanks.